doc/classes/@GlobalScope.xml

```diff
@@ -247,7 +247,7 @@
 				var f = clamp(Vector3i(-7, -8, -9), Vector3i(-1, 2, 3), Vector3i(-4, -5, -6))
 				# f is (-4, -5, -6)
 				[/codeblock]
-				[b]Note:[/b] For better type safety, use [method clampf], [method clampi], [method Vector2.clamp], [method Vector2i.clamp], [method Vector3.clamp], [method Vector3i.clamp], [method Vector4.clamp], [method Vector4i.clamp], or [method Color.clamp].
+				[b]Note:[/b] For better type safety, use [method clampf], [method clampi], [method Vector2.clamp], [method Vector2i.clamp], [method Vector3.clamp], [method Vector3i.clamp], [method Vector4.clamp], [method Vector4i.clamp], or [method Color.clamp]. The methods on vectors and [Color] also clamps each component individually, unlike this method.
```

Comment:
I think the examples with Vector*i above are misleading and should be removed. I don't see any practical use case for using `clamp(Variant, Variant, Variant)` on vector types as the results aren't really intuitive / useful.

The examples `c`, `d`, and `e` seem to show it working similarly to component wise, because the vectors have all components out of the range. But example `e` is just weird. There's no comment explaining this so the examples are bad documentation IMO.

I also think that the sentence added here, while good, still doesn't hit the nail on the head as much as it should.

Reply:
Agreed, some clearer details, maybe outright comparing with the `VectorX.clamp` ones would help?

Yeah the `f` case has the `max` argument be *smaller* than the `min` one, will think about cleaning it up

Reply:
Part of the problem is adding important information in the comments and translations

Will remove the vector cases and think about how to punch up the comment, and put it in a dedicated `Note`

Reply:
How does:
```
[b]Note:[/b] When using this on vectors or [Color] it will [i]not[/i] perform component-wise clamping, and will simply pick [param min] if [code]value &lt; min[/code]. To perform component-wise clamping use the methods above.
```
Sound?

Reply:
Sounds good!